main.m

```diff
@@ -340,6 +340,19 @@ void printAccessibleAttribute(NSString *accessibleString) {
   }
 }
 
+void printSecAccessControl(NSObject *accessControlObject){
+  NSString* accessControlString = accessControlObject.debugDescription;
+  if ([accessControlString rangeOfString:@"pbioc"].location != NSNotFound && [accessControlString rangeOfString:@"pbioh"].location != NSNotFound) {
+    printToStdOut(@"%sSecAccessControl flag: .biometryCurrentSet%s\n", KGRN, KWHT);
+  }else if ([accessControlString rangeOfString:@"pbioc"].location != NSNotFound && [accessControlString rangeOfString:@"pbioh"].location == NSNotFound) {
+    printToStdOut(@"%sSecAccessControl flag: .biometryAny%s\n", KYEL, KWHT);
+  }else if ([accessControlString rangeOfString:@"cpo(DeviceOwnerAuthentication)"].location != NSNotFound) {
+    printToStdOut(@"%sSecAccessControl flag: .userPresence%s\n", KRED, KWHT);
+  }else if ([accessControlString rangeOfString:@"cup(true));odel(true)"].location != NSNotFound) {
+    printToStdOut(@"%sSecAccessControl flag: .devicePasscode%s\n", KRED, KWHT);
```

Comment:
I'm not super familiar with the values of this access control string. Googling, I notice that all of these are listed [here](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags). These line up one to one, except for the "watch" type. Is it worth adding or maybe you don't have access to a watch to figure out what the magic string is to match on for that one?

Reply:
Oh yeah, I must've omitted that as I didn't need that personally. Hopefully I will have more time this week, so I try to commit this and compilation instruction.